src/monocle_apptrace/metamodel/inference_base.py

```diff
@@ -0,0 +1,18 @@
+from monocle_apptrace.metamodel.llm import LLM
+from monocle_apptrace.metamodel.provider import ProviderAttributes
+class Inference:
+    def __init__(self, instance):
+        self.instance = instance
+
+
+    def add_entities(self,span):
+        span.set_attribute("span_type","Inference")
```

Comment:
all the 'keys' that we are adding are in lower case, refer https://github.com/monocle2ai/monocle/blob/main/src/monocle_apptrace/metamodel/entities/entity_types.json

so it would be inference instead of Inference